go.mod

```diff
@@ -12,100 +12,117 @@ replace google.golang.org/grpc => google.golang.org/grpc v1.55.0
 replace github.com/spf13/cobra => github.com/spf13/cobra v1.7.0
 
 require (
-	github.com/ONSdigital/dp-api-clients-go/v2 v2.258.0
-	github.com/ONSdigital/dp-authorisation v0.2.0
+	github.com/ONSdigital/dp-api-clients-go/v2 v2.260.0
+	github.com/ONSdigital/dp-authorisation v0.2.1
 	github.com/ONSdigital/dp-component-test v0.9.0
 	github.com/ONSdigital/dp-elasticsearch/v3 v3.0.1-alpha.4.0.20230308115225-bb7559a89d0c
-	github.com/ONSdigital/dp-healthcheck v1.6.2
+	github.com/ONSdigital/dp-healthcheck v1.6.3
 	github.com/ONSdigital/dp-net/v2 v2.11.2
 	github.com/ONSdigital/dp-otel-go v0.0.6
 	github.com/ONSdigital/dp-search-scrubber-api v0.4.1
 	github.com/ONSdigital/log.go/v2 v2.4.3
 	github.com/cucumber/godog v0.12.6
-	github.com/elastic/go-elasticsearch/v7 v7.10.0
+	github.com/elastic/go-elasticsearch/v7 v7.12.0
 	github.com/google/go-cmp v0.6.0
 	github.com/gorilla/mux v1.8.1
 	github.com/kelseyhightower/envconfig v1.4.0
 	github.com/maxcnunes/httpfake v1.2.4
 	github.com/pkg/errors v0.9.1
 	github.com/smartystreets/goconvey v1.8.1
 	github.com/tdewolff/minify v2.3.6+incompatible
-	go.opentelemetry.io/contrib/instrumentation/github.com/gorilla/mux/otelmux v0.46.1
-	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.46.1
+	go.opentelemetry.io/contrib/instrumentation/github.com/gorilla/mux/otelmux v0.50.0
+	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.50.0
 )
 
 require (
 	github.com/ONSdigital/dp-api-clients-go v1.43.0 // indirect
-	github.com/ONSdigital/dp-mongodb-in-memory v1.5.0 // indirect
+	github.com/ONSdigital/dp-mongodb-in-memory v1.7.0 // indirect
+	github.com/ONSdigital/dp-net v1.5.0 // indirect
```

Comment:
Tracing this use of `github.com/ONSdigital/dp-net` suggests it can be removed:
```
# github.com/ONSdigital/dp-net
(main module does not need module github.com/ONSdigital/dp-net)
```

Reply:
That's interesting... I wonder why it's in the indirect dependencies then.

Reply:
Looks as if the indirect dependency is different, uses V2 like this:
```
# github.com/ONSdigital/dp-net/v2
github.com/ONSdigital/dp-search-api/service
github.com/ONSdigital/dp-net/v2/awsauth
```